src/mono/wasm/build/aot-build.proj

```diff
@@ -0,0 +1,58 @@
+<Project DefaultTargets="BundleTestWasmApp">
```

Comment:
Drop `Directory.Build.{props,targets}` like https://github.com/dotnet/runtime/blob/main/src/tests/BuildWasmApps/Wasm.Build.Tests/WasmBuildAppTest.cs#L620-L660 next to this aot-build.proj on helix. And aot-build.proj doesn't need to setup all the paths, since it will get set up by `WasmApp.LocalBuild*`.